### PR TITLE
Re-init logging system on Windows when using multiprocessing

### LINKED ISSUE #27103: Salt-Minion doesn't display logs for new processes with multiprocessing on
Applies to 2015.5 and up on Windows. Works as expected in Linux.

With multiprocessing turned on all the minion displays is that fact that a job was run:
```
[INFO    ] User root Executing command state.sls with jid 20150914125752187765
[DEBUG   ] Command details {'tgt_type': 'glob', 'jid': '20150914125752187765', '
tgt': 't*', 'ret': '', 'user': 'root', 'arg': ['test.useradd'], 'fun': 'state.sl
s'}
```

Turn multiprocessing off and you get all the details you would expect to see.

### PR DESCRIPTION
**Pull request by cachedout**
Closes #27103

@twangboy Please review this.

### CODE REVIEW

File: salt/minion.py [hunk 1]
```diff
@@ -954,6 +954,13 @@ def _thread_return(cls, minion_instance, opts, data):
         '''
         # this seems awkward at first, but it's a workaround for Windows
         # multiprocessing communication.
+        if sys.platform.startswith('win') and \
+                opts['multiprocessing'] and \
+                not salt.log.is_logging_configured():
+            # We have to re-init the logging system for Windows
+            salt.log.setup_console_logger(log_level=opts.get('log_level', 'info'))
+            if opts.get('log_file'):
+                salt.log.setup_logfile_logger(opts['log_file'], opts.get('log_level', 'info'))
```
Comment (twangboy) [inline salt/minion.py]: @cachedout Shouldn't the second parameter be `log_level_logfile` since it's setting up the log file itself?

**Comment by twangboy** [comment]
@cachedout Tested successfully on my minion.

File: salt/minion.py (hunk 1 above)
Comment (cachedout) [inline salt/minion.py]: Yes. Thanks. Fixed.

**Comment by cachedout** [comment]
@twangboy Did you need this in 2015.5 as well or just 2015.8?

**Comment by cachedout** [comment]
@meggiebot Thanks. Merged.

**Pull request merged**